dist/vl-proza-message.js

```diff
@@ -153,17 +153,19 @@ export class VlProzaMessage extends vlElement(HTMLElement) {
   }
 
   _loadMessage() {
-    if (this._domain && this._code) {
-      VlProzaMessage._getMessage(this._domain, this._code).then((message) => {
-        this._wysiwygElement.innerHTML = message;
-        this.__wrapWysiwygElement();
-        if (this.__containsBlockElement(message)) {
-          this.setAttribute('data-vl-block', '');
-        }
-      });
-    } else {
-      this._wysiwygElement.innerHTML = null;
-    }
+    awaitUntil(() => this._wysiwygElement).then(() => {
```

Comment:
enkel een beetje jammer dat we hier expliciet moeten wachten. Er is geen event waar we kunnen naar luisteren?

Reply:
Nee, het wysiwyg element wordt door ons aangemaakt in de connected callback dus is volledig eigen beheer. Wachten op zich is eigenlijk ook niet nodig, maar bij unit testen kan het zijn dat de connected callback nog niet aangeroepen werd en er al http mocks geflushed worden waardoor er problemen onstaan. Puur defensief programmeren uitbreiding voor de unit testen, alternatief zou zijn dat jullie de unit testen aanpassen.